### PR TITLE
Feat/remove system icons

### PR DESCRIPTION
**Pull request by ferreroltd**
last badge of icons migration

### CODE REVIEW

File: packages/apps/graph-client/src/components/common/loader/loader.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { SystemIcon } from '@kadena/react-ui';
+import { MonoLoading } from '@kadena/react-icons';
 import React from 'react';
 import { spin } from './styles.css';
 
 const Loader: React.FC = () => {
-  return <SystemIcon.Loading className={spin} size="lg" />;
+  // add bigger size wrapper
```
Comment (r-mulder) [inline packages/apps/graph-client/src/components/common/loader/loader.tsx:6]: Do you still need to do something here?

File: packages/libs/react-ui/src/components/Button/Button.stories.tsx [hunk 2]
```diff
@@ -68,6 +68,13 @@ const meta: Meta<IButtonProps> = {
         type: 'boolean',
       },
     },
+    avatarProps: {
+      description:
+        'Props for the avatar component which can be rendered instead of startIcon',
+      control: {
+        type: 'boolean',
+      },
+    },
```
Comment (r-mulder) [inline packages/libs/react-ui/src/components/Button/Button.stories.tsx:77]: Same as the other MR, this is removed from the button.

File: packages/apps/graph-client/src/components/error-box/error-box.tsx [hunk 3]
```diff
@@ -33,7 +34,7 @@ export const ErrorBox = (props: IErrorBoxProps): JSX.Element => {
   }
 
   return (
-    <Notification intent="negative" icon={<SystemIcon.Close />} role="status">
+    <Notification intent="negative" icon={<MonoClose />} role="status">
```
Comment (nil-amrutlal-dept) [inline packages/apps/graph-client/src/components/error-box/error-box.tsx:37]: image

I am not sure if it was already, but this icon is not aligned. If its outside your scope please ignore this :)

File: packages/apps/graph-client/src/pages/transactions/[key].tsx [hunk 4]
```diff
@@ -116,7 +116,7 @@ const RequestKey: React.FC = () => {
                     transaction.result.goodResult && (
                       <Notification
                         intent="positive"
-                        icon={<SystemIcon.Check />}
+                        icon={<MonoCheck />}
```
Comment (nil-amrutlal-dept) [inline packages/apps/graph-client/src/pages/transactions/[key].tsx:119]: image

Same here

File: packages/apps/graph-client/src/components/error-box/error-box.tsx (hunk 3 above)
Comment (nil-amrutlal-dept) [inline packages/apps/graph-client/src/components/error-box/error-box.tsx:37]: image

I am not sure if it was already, but this icon is not aligned. If its outside your scope please ignore this :)

File: packages/apps/graph-client/src/pages/transactions/[key].tsx (hunk 4 above)
Comment (nil-amrutlal-dept) [inline packages/apps/graph-client/src/pages/transactions/[key].tsx:119]: image

Same here

Comment (ferreroltd) [reply packages/apps/graph-client/src/pages/transactions/[key].tsx:119]: btw fixing this in the notification v2

**Review by sanderlooijenga — APPROVED** [review]

File: packages/apps/docs/src/components/Layout/components/Header/Header.tsx [hunk 5]
```diff
@@ -65,15 +65,15 @@ export const Header: FC<IProps> = ({ menuItems, layout = 'full' }) => {
             href="https://twitter.com/kadena_io"
             title="Go to our X"
           >
-            <SystemIcon.Twitter />
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Layout/components/Header/Header.tsx:68]: image

icon sizes seem smaller. is this correct, design wise?

Comment (ferreroltd) [reply packages/apps/docs/src/components/Layout/components/Header/Header.tsx:68]: checked with isa and its correct @sstraatemans

**Review by sstraatemans — APPROVED** [review]

**Pull request merged**